src/features/HOC/withConfirmation/ui/index.tsx

```diff
@@ -35,7 +35,7 @@ export const withConfirmation = ({text, positive, negative, onPositiveClick, onN
       }, []);
 
       return (
-        <div>
+        <>
```

Comment:
nit: why this change?) I usually see such a change when there is some dirty css tricks... just want to make sure we do not do that

Reply:
restructured, looks like working with div